libs/auc_analysis.py

```diff
@@ -200,6 +200,7 @@ def summary_analysis_binary(miss_summary_file, summary_file, roc_fig,
     plt.title('ROC curve')
     plt.legend(loc="lower right")
     plt.savefig(roc_fig)
+    plt.clf()
```

Comment:
```suggestion
    plt.close()
```

`clf`より`close`の方が良いのではないかと言われているようです。
cf. [matplotlibで、could not allocate memoryが出たときの解決法: 長続き目指しブログ](http://naga-tsuzuki.sblo.jp/article/176236263.html)
cf. [When to use cla(), clf() or close() for clearing a plot in matplotlib? - Stack Overflow](https://stackoverflow.com/questions/8213522/when-to-use-cla-clf-or-close-for-clearing-a-plot-in-matplotlib)
> In short if memory is a concern use plt.close(fig)

windowを再利用することはないと思うので`close`で良いのではないかと思いますがいかがでしょうか。